R/options.R

```diff
@@ -123,3 +126,35 @@ sass_options <- function(
   class(ret) <- c("sass_options", class(ret))
   ret
 }
+
+#' @rdname sass_options
+#' @param ... arguments to [sass_options()]. If specified, these options will take priority over any options set globally (or via `shiny::devmode()`)
+#' @importFrom utils modifyList
+#' @export
+sass_options_get <- function(...) {
+
+  global_opts <- get_shiny_devmode_option(
+    "sass.options",
+    default = sass_options(),
+    devmode_default = sass_options(
```

Comment:
If `sass_options_set()` has been called, then these devmode settings will be ignored.